net/quagga/src/opnsense/mvc/app/views/OPNsense/Quagga/diagnosticsospf.volt

```diff
@@ -242,19 +313,60 @@ TODO
         <td><%= entry["cost"] %></td>
         <td><%= entry["area"] %></td>
         <td><%= entry["tag"] %></td>
-        <td><%= entry["via"] %></td>
+        <td><%= translate(entry["via"]) %></td>
         <td><%= entry["via_interface"] %></td>
       </tr>
     <% }); %>
   </tbody>
 </table>
 </script>
+<script type="text/x-template" id="neighbortpl">
+  <table>
+    <thead>
+      <tr>
+        <th>{{ lang._('Neighbor ID') }}</th>
+        <th>{{ lang._('Priority') }}</th>
+        <th>{{ lang._('State') }}</th>
+        <th>{{ lang._('Dead Time') }}</th>
+        <th>{{ lang._('Address') }}</th>
+        <th>{{ lang._('Interface') }}</th>
+        <th>{{ lang._('RXmtL') }}</th>
+        <th>{{ lang._('RqstL') }}</th>
+        <th>{{ lang._('DBsmL') }}</th>
```

Comment:
@fichtner I don't know what this strings mean - so I output them as I get them from quagga.